migrations/tenant/0016-note-teams@add-auto-increment-id.sql

```diff
@@ -6,7 +6,7 @@ CREATE SEQUENCE IF NOT EXISTS public.note_teams_id_seq
     NO MAXVALUE
     CACHE 1;
 
-ALTER TABLE public.note_teams_id_seq OWNER TO codex;
+-- ALTER TABLE public.note_teams_id_seq OWNER TO codex;
```

Comment:
@slaveeks, in this migration error raises, that we cannot change owner of sequence "note_teams_id_seq". Check this please.